src/v/config/node_config.cc

```diff
@@ -149,73 +155,69 @@ node_config::node_config() noexcept
   , recovery_mode_enabled(
       *this,
       "recovery_mode_enabled",
-      "If true, start redpanda in \"metadata only\" mode, skipping "
-      "loading user partitions and allowing only metadata operations.",
+      "If `true`, start Redpanda in recovery mode, where user partitions are not loaded and only administrative operations are allowed.",
       {.visibility = visibility::user},
       false)
   , storage_failure_injection_config_path(
       *this,
       "storage_failure_injection_config_path",
-      "Path to the configuration file used for low level storage failure "
-      "injection",
+      "Path to the configuration file used for low level storage failure injection.",
       {.visibility = visibility::tunable},
       std::nullopt)
   , verbose_logging_timeout_sec_max(
       *this,
       "verbose_logging_timeout_sec_max",
-      "Maximum duration in seconds for verbose (i.e. TRACE or DEBUG) logging. "
-      "Values configured above this will be clamped. If null (the default) "
-      "there is no limit. Can be overridded in the Admin API on a per-request "
-      "basis.",
+      "Maximum duration in seconds for verbose (`TRACE` or `DEBUG`) logging. Values configured above this will be clamped. If null (the default) there is no limit. Can be overridden in the Admin API on a per-request basis.",
       {.visibility = visibility::tunable},
       std::nullopt,
       {.min = 1s})
   , fips_mode(
       *this,
       "fips_mode",
-      "Controls whether Redpanda starts in FIPS mode.  This property "
-      "allows for three values: 'disabled', 'enabled', and 'permissive'.  With "
-      "'enabled', Redpanda first verifies that the operating "
-      "system "
-      "is enabled for FIPS by checking /proc/sys/crypto/fips_enabled.  If the "
-      "file does not exist or does not return '1', Redpanda immediately "
-      "exits.  With 'permissive', the same check is performed "
-      "but a WARNING is logged and Redpanda continues to run.  After "
-      "the check is complete, Redpanda loads the "
-      "OpenSSL FIPS provider into the OpenSSL library.  After this is "
-      "complete, Redpanda is operating in FIPS mode, which means that the "
-      "TLS cipher suites available to users are limited to TLSv1.2 "
-      "and TLSv1.3, and of those, only the ones that use NIST-approved "
-      "cryptographic methods.  For more information about FIPS, refer to "
-      "Redpanda documentation.",
+      "Controls whether Redpanda starts in FIPS mode.  This property allows for three values: 
+
+* Disabled - Redpanda does not start in FIPS mode.
+
+* Permissive - Redpanda performs the same check as enabled, but a warning is logged, and Redpanda continues to run. Redpanda loads the OpenSSL FIPS provider into the OpenSSL library. After this completes, Redpanda is operating in FIPS mode, which means that the TLS cipher suites available to users are limited to the TLSv1.2 and TLSv1.3 NIST-approved cryptographic methods.
```

Comment:
* we try to line wrap around 80 columns
* maybe we need to maintain the human formatted version somewhere else?

Reply:
It would be awesome if we could move those messages to a message bundle of sorts.